helm/prometheus-rules/templates/recording-rules/gs-managed-app-deployment-status.rules.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: gs-managed-app-deployment-status.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: gs-managed-app-deployments
+    rules:
+    - expr: kube_deployment_status_replicas_available *
+            on (deployment) group_left(label_giantswarm_io_service_type, label_app)
+            kube_deployment_labels{label_giantswarm_io_service_type="managed"}
+      record: kube_deployment_status_replicas_available_gs
```

Comment:
Would it not make more sense to call this managed_app_deployment_status_replicas_available ?

Reply:
yeah good input! Thanks, will change!